pkg/reporter/stdout_reporter.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2024 The Authors (see AUTHORS file)
```

Comment:
We may want to name this "local" instead of stdout.

Reply:
I went back and forth. Went literal witht he reporter naming. Was planning on introducing a -reporter=local flag, and it will create a stdout reporter. Tomato tomato?